### PR TITLE
Use TTY::Cursor.hide instead of constants

### PR DESCRIPTION
**Pull request by benklop**
### Describe the change
fixes issue #38 
### Why are we doing this?
crashing is not desirable

### CODE REVIEW

**Comment by piotrmurach** [comment]
Hi,

Thanks for contributing! 

These constants are remnants from my refactorings. In this case, they should be replaced by [tty-cursor](https://github.com/piotrmurach/tty-cursor), you can see it [here](https://github.com/piotrmurach/tty-spinner/blob/master/lib/tty/spinner.rb#L358).

**Comment by piotrmurach** [comment]
Hey! Thanks for updating your PR. Would it possible for yourself to add a test for this as well so I don't break things in the future? That would be a cherry on a cake!

**Comment by benklop** [comment]
I'll take a look at creating a test, but I have some other work to complete first

File: spec/unit/spin_spec.rb [hunk 1]
```diff
@@ -46,26 +46,55 @@
     ].join)
   end
 
-  it "can spin and redraw indent" do
-    multi_spinner = double("MultiSpinner")
-    allow(multi_spinner).to receive(:synchronize).and_yield
-    allow(multi_spinner).to receive(:next_row).and_return(1)
-    allow(multi_spinner).to receive(:rows).and_return(1)
-    allow(multi_spinner).to receive(:line_inset).and_return("--- ")
+  context "with the cursor" do
```
[the remark this answers is not in the document]
Comment (benklop) [reply spec/unit/spin_spec.rb]: if you can't have longer blocks, how do you use context blocks?

**Comment by benklop** [comment]
@piotrmurach i'm not really used to the style you're using for your specs, hopefully context blocks are acceptable

File: spec/unit/spin_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (piotrmurach) [reply spec/unit/spin_spec.rb]: Don't worry about these warnings.

**Comment by piotrmurach** [comment]
Thanks for making this change and adding tests! 🙇

**Pull request merged**